libsentrykube/events.py

```diff
@@ -83,6 +83,9 @@ def report_terragrunt_event(
     else:
         raise RuntimeError("Unable to determine what slice you're running in.")
 
+    if region == "us":
+        region = "saas"
+
```

Comment:
Are we porting over this mapping in this external repo or fixing it?

Reply:
Porting this over. Till this repo gets merged into ops, no fixes are happening on this repo. That makes diffing of the 2 directories being ported over easy.